4_processing_features/README.md

```diff
@@ -0,0 +1,27 @@
+# 4. Processing Extracted Single Cell Features 
+
+In this module, we present our pipeline for processing outputted `.sqlite` file with single cell features from CellProfiler.
+The processed features are saved into compressed `.csv.gz` for use during statistical analysis.
+
+## Pycytominer
+
+We use [Pycytominer](https://github.com/cytomining/pycytominer) to perform the aggregation, merging, and normalization of the NF1 single cell features.
+
+For more information regarding the functions that we used, please see [the documentation](https://pycytominer.readthedocs.io/en/latest/pycytominer.cyto_utils.html#pycytominer.cyto_utils.cells.SingleCells.merge_single_cells) from the Pycytominer team.
+
+### Normalization
+
+Normalization of the data is important because there will be variety in the shapes of distributions. To make statsitical analysis easier, we normalize using standardized method.
+
+---
+
+## Step 1: Setup Processing Feature Environment
```

Comment:
I suppose that we'll add the next steps in a future PR?

Although maybe it belongs here? What else would belong in this module?

Reply:
@gwaybio 

I guess what do you mean by next steps? I just added what I understood was going on. 

Would we want to add steps for performing statistics or do you mean something else?

Reply:
Oh... you are right lol

Reply:
My bad, brain was not thinking about that! I will add that and make the commit when I am back! :smile: